resources/carrier/en/380.txt

```diff
@@ -13,19 +13,19 @@
 # limitations under the License.
 
 # Data taken from:
-# https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000DB0004PDFE.pdf
+# https://www.itu.int/oth/T02020000DB/en
 
 # Telesystems of Ukraine is commonly known as PEOPLEnet.
 
 38050|Vodafone
-38063|Life
+38063|Lifecell
```

Comment:
One last correction and then we are good to go :)

As per https://www.lifecell.ua/en/about_lifecell/main/ it should be in lower case so can you change 'Lifecell' to 'lifecell'